packages/cli/src/user-management/email/templates/user-invited.mjml

```diff
@@ -0,0 +1,18 @@
+<mjml>
+	<mj-include path="./_common.mjml" />
+	<mj-body>
+		<mj-section>
+			<mj-column>
+				<mj-text font-size="24px" color="#ff6f5c"> Welcome to n8n! 🎉 </mj-text>
+			</mj-column>
+		</mj-section>
+		<mj-section background-color="#FFFFFF" border="1px solid #ddd">
+			<mj-column>
+				<mj-text>You have been invited to join n8n: {{domain}}.</mj-text>
```

Comment:
Just in case the appended dot might cause some email clients not to detect and render the URL:

```suggestion
				<mj-text>You have been invited to join n8n: {{domain}}</mj-text>
```